### DISCUSSION TITLE
Animated Objects in 3D environment Bonvision

### DISCUSSION THREAD

**Discussion by KostasTsar** [bonsai-rx/bonsai#1264] [Q&A]
Hello,

I was wondering if it's currently possible to load an 3d model with an animation in a bonvision 3D environment (e.g. an animated blender model). If that is not currently implemented, is there a plan to do it? Thank you in advance.

**Comment by chiyu1203** [comment]
i am interested in this topic too! I have only watched [tutorials](https://www.youtube.com/watch?v=4Ypo0BCW36Y&t=1449s) to load inanimated object from Blender but not animated model yet.
